layer/layer_utils.h

```diff
@@ -53,6 +55,28 @@
       reinterpret_cast<PFN_vk##FUNC_NAME_>(gdpa(*device, "vk" #FUNC_NAME_))
 
 namespace performancelayers {
+// steady_clock, system_clock, and high_resolution_clock comparison:
+// 1. steady_clock: A monotonic clock. The current value of steady_clock does
+// not matter, but the guarantee that is strictly increasing is useful for
+// calculating the difference between two samples.
+// 2. system_clock: Is the wall-clock. Used for time/day related stuff.
+// 3. high_resolution_clock: An alias for one of the above.
+
+// Returns a system_clock::time_point as the timestamp. The system_clock acts
+// like a real clock and shows the current time, while the steady_clock is a
+// monotonic clock and behaves like a chronometer. This is why system_clock is
+// used for the timestamp.
+std::chrono::system_clock::time_point GetTimestamp();
```

Comment:
Consider introducing aliases for these clocks here that layers can use.
```c++
using TimestampClock = std::chrono::system_clock;
using DurationClock = std::chrono::steady_lock;
```

I think this would help ensuring that layers use the appropriate clocks.